src/main/java/xyz/nucleoid/plasmid/game/StartResult.java

```diff
@@ -6,38 +6,64 @@
 
 import javax.annotation.Nullable;
 
+/**
+ * Describes the result of a {@link GameWorld} attempting to start.
+ */
 public final class StartResult {
+    public static final StartResult ALREADY_STARTED = error(new LiteralText("This game has already started!"));
```

Comment:
It's a stylistic thing, but personally prefer to keep the consistency of having them all fields/all methods.

Reply:
Are you saying I should keep the fields private and offer methods for returning them (so similar to before, but without an object being created on every call)?

Reply:
Yea- either that or can make `OK` a field also

Reply:
I think fields make more sense, only because it helps to suggest (while autocompleting) that you can also make your own (and that the fields are just templates). I think I'm going to refactor `JoinResult`/`StartResult` in a separate PR later anyways, so I left `JoinResult` as-is for now.